src/components/wallet/methods/updateAccount.ts

```diff
@@ -7,19 +7,32 @@ import {
 import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
 
+async function claimableBalanceQuery(wallet: Wallet) {
+  try {
+    let resp = await wallet.server
+      .claimableBalances()
+      .claimant(wallet.account.publicKey)
+      .call()
+    resp.records.forEach((record) => {
+      wallet.logger.response(
+        'Claimable Balances Available ',
+        loOmit(record, ['_links', 'paging_token', 'self'])
+      )
+    })
+  } catch (err) {
+    wallet.error = handleError(err)
+  }
+}
+
 export default async function updateAccount(this: Wallet) {
   try {
     this.error = null
     this.loading = { ...this.loading, update: true }
-    // this.logger.instruction('Updating account...')
     await this.server
       .accounts()
       .accountId(this.account.publicKey)
       .call()
       .then((account) => {
```

Comment:
Can we adjust this call too? I'm a bit surprised this even works. If you use `await`, that means you're expecting the code that follows to return a JS `Promise`. But this call ends with a `then()` clause that I don't think returns a `Promise` since its not marked as async like so:
```
.then(async (account) => {
  ...   
}
```

Reply:
So maybe this actually works, idk, but it looks weird and it doesn't match the pattern you now follow in `claimableBalanceQuery()`

Reply:
Take a look at this site: https://javascript.info/async-await

See how they replace ALL callbacks (`.finally()`, `.then()`, and `.catch()`)? I'd like use to move to this style so we can avoid thinking about what kind of error catching we need (`try {} catch {}` vs `.then() and .catch()`

Reply:
Its really just a lot easier to read and understand JS when it doesn't have crazy callback chains

Reply:
I think the changes I made align more with your change request.
It does make the code more clean!